src/pkg/quota/util.go

```diff
@@ -32,7 +32,7 @@ func IsSafe(hardLimits types.ResourceList, currentUsed types.ResourceList, newUs
 			continue
 		}
 
-		if hardLimit == types.UNLIMITED || value == currentUsed[resource] {
```

Comment:
@lengrongfu You removed the condition `value == currentUsed[resource]`, could you explain the reason for this?